src/index.html

```diff
@@ -15,8 +15,39 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="container">
+        <a
+          href="https://th.bing.com/th/id/OIP.axjC06-qxFXElNPCbM1MPAAAAA?rs=1&pid=ImgDetMain"
+          target="_blank"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo-moyo"
+          />
+        </a>
+        <div class="nav-bar">
+          <div class="nav-link is-active">Apple</div>
+          <div class="nav-link is-active">Samsung</div>
+          <div class="nav-link is-active">smartphones</div>
+          <div
+            class="nav-link is-active"
+            data-qa="hover"
+          >
+            laptops & computers
```

Comment:
Lines of code should have 80 characters max. The text 'laptops & computers' is too long and should be broken into multiple lines if necessary.